### ISSUE TITLE
Example 2 ... we never stop recording ...?

### ISSUE THREAD

**Issue by kolban-google** [dialogflow/selfservicekiosk-audio-streaming#16]
Looking at the example called example2.html, I see that clicking the stop button toggles the enabled status but I am not seeing it change anything related to the RecordRTC object and hence I don't think recording is stopped.

**Comment by nehal2210** [comment]
Yes, There is no Implementation on the stop streaming button, also guide us how to stop the session of the detectIntentStream.
